backend/app/db/crud/document.py

```diff
@@ -313,65 +315,7 @@ def get_document_detail(db, document_id) -> DocumentDetailResponse:
     ).all()
 
     # Get all associated documents
-    # start with all documents with a reference from this document_id
-    related_documents_to = set(
-        DocumentOverviewResponse(
-            document_id=d.id,
-            name=d.name,
-            description=d.description,
-            country_code=g.value,
-            country_name=g.display_value,
-            publication_ts=d.publication_ts,
-        )
```

Comment:
is now the right time to remove this? it feels like switching document `GET` to using the new table should happen after we've migrated the data... otherwise we temporarily lose access to all related document information...

Reply:
re-instated all the association code - to be removed later :smile: